crates/red_knot_python_semantic/resources/mdtest/expression/attribute.md

```diff
@@ -29,6 +29,6 @@ def _(flag: bool):
     # error: [possibly-unbound-attribute] "Attribute `possibly_unbound` on type `Literal[A]` is possibly unbound"
     reveal_type(A.possibly_unbound)  # revealed: str
 
-    # error: [unresolved-attribute] "Type `Literal[A]` has no attribute `non_existent`"
+    # error: [unresolved-attribute] "Attribute `non_existent` can only be accessed on instances of type `Literal[A]`, not on the class object itself."
```

Comment:
This error message change looks like a regression that should be reversed. The new error message is misleading because it implies that accessing `non_existent` on the instance type would work, when it would not, because the attribute doesn't exist at all. (I think maybe this is also discussed in another comment thread below?)

Reply:
Oops ignore, commented before you made the update.